R/kenwardroger.R

```diff
@@ -35,14 +35,12 @@ h_get_kr_comp <- function(tmb_data, theta) {
 #'
 #' @param object (`mmrm`)\cr object.
 #' @param contrast (`matrix`)\cr contrast matrix.
-#' @param linear (`logical`)\cr whether to use linear Kenward-Roger approximation.
 #'
 #' @return List with `num_df`, `denom_df`, `f_stat` and `p_val` (2-sided p-value).
 #'
 #' @keywords internal
-h_df_md_kr <- function(object, contrast, linear = FALSE) {
+h_df_md_kr <- function(object, contrast) {
```

Comment:
linear option removed because d.f. is the same for two methods, and beta_vcov is different. but beta_vcov is adjusted in `mmrm` call

Reply:
sounds good